### ISSUE TITLE
MPICH3 support

### ISSUE THREAD

**Issue by GoogleCodeExporter** [adda-team/adda#203]
```
Users of mpich2 are expected to upgrade to mpich3 over time. In principle, ADDA 
should work with mpich3 with no changes. However, certain informational 
improvements are required, like showing the version of MPICH3 providing 
installation instructions of it. The latter is especially relevant for Windows.
```

Original issue reported on code.google.com by `yurkin` on 28 Jul 2014 at 11:15

**Comment by myurkin** [comment]
Actually, we will use MS_MPI on Windows. This issue will be fixed with #246 .

**Comment by myurkin** [comment]
Fixed with commit afcfc94 . Installation instructions are present in https://github.com/adda-team/adda/wiki/InstallingMPI .
